classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_studnt(student)
+    @student.push?(student)
```

Comment:
- [ ] please remove the '?' in this line as it will make your application not work.

Reply:
done